pkg/scheduler/frameworkext/framework_extender_test.go

```diff
@@ -85,12 +85,12 @@ func (h *TestTransformer) AfterPreFilter(ctx context.Context, cycleState *framew
 	return nil
 }
 
-func (h *TestTransformer) BeforeFilter(ctx context.Context, cycleState *framework.CycleState, pod *corev1.Pod, nodeInfo *framework.NodeInfo) (*corev1.Pod, *framework.NodeInfo, bool, *framework.Status) {
+func (h *TestTransformer) BeforeFilter(ctx context.Context, cycleState *framework.CycleState, pod *corev1.Pod, nodeInfo *framework.NodeInfo) (*corev1.Pod, *framework.NodeInfo, *framework.CycleState, bool, *framework.Status) {
```

Comment:
Very weird. Why does it need to return a new CycleState here?

Reply:
done